browser/package.json

```diff
@@ -14,17 +14,19 @@
   "devDependencies": {
     "@babel/preset-env": "^7.18.6",
     "@babel/preset-react": "^7.18.6",
+    "@parcel/transformer-sass": "^2.6.2",
     "babel-jest": "^28.1.2",
     "jest": "^28.1.2",
     "parcel": "^2.6.2",
+    "path-browserify": "^1.0.1",
```

Comment:
What does path-browserify do?

Reply:
That's a good question 😅 It might have been something that snuck in when I was setting up sass stuff. I'll look into it.

Reply:
Ah I think this was when I was experimenting with how to get the path of the icon .svg location and this got installed, but I don't think it's actually needed so I'm getting rid of it.